commit f4c743e2ef079bfe288d37d9056a8827c684d01e
Author: Dominique Leuenberger <dimstar@opensuse.org>
Date:   Thu Dec 1 22:08:12 2011 +0100

    Bug 665335: Only #include <glib.h>: the other headers are implicit

diff --git a/objects/custom/shape_typeinfo.c b/objects/custom/shape_typeinfo.c
index c6133b1..3fb3d73 100644
--- a/objects/custom/shape_typeinfo.c
+++ b/objects/custom/shape_typeinfo.c
@@ -27,7 +27,7 @@
 #include "custom_util.h"
 #include <string.h>
 #include <stdarg.h>
-#include <glib/gstrfuncs.h>
+#include <glib.h>
 #include <glib/gstdio.h>
 #include <libxml/parser.h>
 
diff --git a/plug-ins/pgf/render_pgf.c b/plug-ins/pgf/render_pgf.c
index f5c3618..16ec5e1 100644
--- a/plug-ins/pgf/render_pgf.c
+++ b/plug-ins/pgf/render_pgf.c
@@ -61,7 +61,7 @@ TODO:
 #endif
 #include <errno.h>
 
-#include <glib/gprintf.h>
+#include <glib.h>
 #include <glib/gstdio.h>
 
 #include "intl.h"
diff --git a/tests/test-boundingbox.c b/tests/test-boundingbox.c
index d9bbdba..8b3111d 100644
--- a/tests/test-boundingbox.c
+++ b/tests/test-boundingbox.c
@@ -28,8 +28,6 @@
 #include <glib.h>
 #include <glib-object.h>
 
-#include <glib/gtestutils.h>
-
 #include "dialib.h"
 
 /*
diff --git a/tests/test-objects.c b/tests/test-objects.c
index 957fa0f..6fcb438 100644
--- a/tests/test-objects.c
+++ b/tests/test-objects.c
@@ -28,8 +28,6 @@
 #include <glib.h>
 #include <glib-object.h>
 
-#include <glib/gtestutils.h>
-
 #include "object.h"
 #include "plug-ins.h"
 #include "dialib.h"
