diff --git a/cmake/Modules/FindLibClang.cmake b/cmake/Modules/FindLibClang.cmake
index 529ab0a..9c699d0 100644
--- a/cmake/Modules/FindLibClang.cmake
+++ b/cmake/Modules/FindLibClang.cmake
@@ -5,6 +5,10 @@ if (UNIX)
                      HINTS  
                      /usr/lib 
                      /usr/local/lib 
+                     /usr/lib/llvm-10.0/lib
+                     /usr/lib/llvm-9.0/lib
+                     /usr/lib/llvm-8.0/lib
+                     /usr/lib/llvm-7.0/lib
                      /usr/lib/llvm-6.0/lib
                      /usr/lib/llvm-5.0/lib
                      /usr/lib/llvm-4.2/lib
@@ -24,8 +27,12 @@ if (UNIX)
 
         find_path(LIBCLANG_INCLUDE_T NAMES clang-c/Index.h
                   HINTS 
-                  /usr/lib/llvm-6.0/include 
-                  /usr/lib/llvm-5.0/include 
+                  /usr/lib/llvm-10.0/include
+                  /usr/lib/llvm-9.0/include
+                  /usr/lib/llvm-8.0/include
+                  /usr/lib/llvm-7.0/include
+                  /usr/lib/llvm-6.0/include
+                  /usr/lib/llvm-5.0/include
                   /usr/lib/llvm-4.2/include 
                   /usr/lib/llvm-4.1/include 
                   /usr/lib/llvm-4.0/include 
diff --git a/cmake/Modules/FindLibLLDB.cmake b/cmake/Modules/FindLibLLDB.cmake
index 794063b..af091a5 100644
--- a/cmake/Modules/FindLibLLDB.cmake
+++ b/cmake/Modules/FindLibLLDB.cmake
@@ -6,6 +6,10 @@ if (UNIX)
                      HINTS  
                      /usr/lib 
                      /usr/local/lib 
+                     /usr/lib/llvm-10.0/lib
+                     /usr/lib/llvm-9.0/lib
+                     /usr/lib/llvm-8.0/lib
+                     /usr/lib/llvm-7.0/lib
                      /usr/lib/llvm-6.0/lib
                      /usr/lib/llvm-5.0/lib
                      /usr/lib/llvm-4.2/lib
@@ -25,6 +28,10 @@ if (UNIX)
 
         find_path(LIBLLDB_INCLUDE_T NAMES lldb/API/SBDebugger.h
                   HINTS 
+                  /usr/lib/llvm-10.0/include
+                  /usr/lib/llvm-9.0/include
+                  /usr/lib/llvm-8.0/include
+                  /usr/lib/llvm-7.0/include
                   /usr/lib/llvm-6.0/include
                   /usr/lib/llvm-5.0/include
                   /usr/lib/llvm-4.2/include 
