From e3c45800a7ea6a671c5bcfc6253870703ee9a454 Mon Sep 17 00:00:00 2001
From: Dirk Schoebel <dirk@liji.de>
Date: Sun, 26 Mar 2023 23:15:00 +0200
Subject: [PATCH] don't override c++ version set by build system

---
 DebugAdapterClient/CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/DebugAdapterClient/CMakeLists.txt b/DebugAdapterClient/CMakeLists.txt
index 93d69c0..529217b 100644
--- a/DebugAdapterClient/CMakeLists.txt
+++ b/DebugAdapterClient/CMakeLists.txt
@@ -1,5 +1,4 @@
 # lldb requires C++11
-add_definitions(-std=c++11)
 set(PLUGIN_NAME "DebugAdapterClient")
 project(DebugAdapterClient)
 
-- 
2.40.0

